packages/synapse-react-client/src/components/ChallengeTeamWizard/ChallengeTeamWizard.stories.tsx

```diff
@@ -1,16 +1,37 @@
 import { Meta, StoryObj } from '@storybook/react'
 import ChallengeTeamWizard from './ChallengeTeamWizard'
+import mockProject from '../../mocks/entity/mockProject'
 
 const meta = {
   title: 'Synapse/Challenge/ChallengeTeamWizard',
   component: ChallengeTeamWizard,
+  argTypes: {
+    isAuthenticated: {
+      control: { type: 'boolean' },
+      defaultValue: true,
+    },
+  },
 } satisfies Meta
+
 export default meta
+
 type Story = StoryObj<typeof meta>
 
 export const Demo: Story = {
   args: {
-    projectId: 'syn51565244',
+    projectId: mockProject.id,
     isShowingModal: true,
   },
+  parameters: {
+    stack: 'mock',
+  },
+}
```

Comment:
Add a mock story so we can click through this without registering for a challenge in prod